app/Domain/Goalcanvas/Templates/canvasDialog.blade.php

```diff
@@ -0,0 +1,182 @@
+@extends($layout)
+@section('content')
+
+@php
+
+$canvasName = 'goal';
+
+$canvasItem = $tpl->get('canvasItem');
+$canvasTypes = $tpl->get('canvasTypes');
+$hiddenStatusLabels = $tpl->get('statusLabels');
+$statusLabels = $statusLabels ?? $hiddenStatusLabels;
+$hiddenRelatesLabels = $tpl->get('relatesLabels');
+$relatesLabels = $relatesLabels ?? $hiddenRelatesLabels;
+$dataLabels = $tpl->get('dataLabels');
+
+$id = "";
+if (isset($canvasItem['id']) && $canvasItem['id'] != '') {
+$id = $canvasItem['id'];
+}
+
+$currentCanvas = $tpl->get('currentCanvas');
+
+if (isset($_GET['canvasId'])) {
+$currentCanvas = (int)$_GET['canvasId'];
+}
+
+@endphp
+
+<script type="text/javascript">
+    window.onload = function() {
+        if (!window.jQuery) {
+            //It's not a modal
+            location.href = "<?= BASE_URL ?>/<?= $canvasName ?>canvas/showCanvas?showModal=<?php echo $canvasItem['id']; ?>";
+        }
+    }
+</script>
+
+<div style="width:1000px">
+
+    <h1><i class="fas <?= $canvasTypes[$canvasItem['box']]['icon']; ?>"></i> <?= $canvasTypes[$canvasItem['box']]['title']; ?></h1>
+
+    {{ $tpl->displayNotification() }}
+
+    <form class="formModal" method="post" action="{{ BASE_URL."/$canvasName/canvas/editCanvasItem/$id" }}">
+
+        <input type="hidden" value="{{ $currentCanvas }}" name="canvasId">
+        <input type="hidden" value="{{ $canvasItem['box'] }}" name="box" id="box">
+        <input type="hidden" value="{{ $id }}" name="itemId" id="itemId">
+        <input type="hidden" name="milestoneId" value="{{ $canvasItem['milestoneId'] ?? '' }}">
+        <input type="hidden" name="changeItem" value="1">
+
+        <div class="col-md-8">
+            <label>{{ __("label.what_is_your_goal") }}</label>
+            <input type="text" name="title" value="{{ $canvasItem['title'] }}" style="width:100%"><br>
+
+            @if (!empty($relatesLabels))
+            <label>{{ __("label.relates") }}</label>
+            <select name="relates" style="width: 50%" id="relatesCanvas">
+            </select><br>
+            @else
+            <input type="hidden" name="relates" value="{{ $canvasItem['relates'] ?? array_key_first($hiddenRelatesLabels) }}">
+            @endif
+            <br>
+            <h4 class="widgettitle title-light" style="margin-bottom:0px;"><i class="fa-solid fa-ranking-star"></i> {{ __("Metrics") }}</h4>
+
+            @php
+
+            $tpl->dispatchTplEvent('beforeMeasureGoalContainer', $canvasItem); @endphp
+            <div id="measureGoalContainer">
+                <label>How will you measure this objective. What metric will you be using.</label>
```

Comment:
While we are in the files: Can you move any english content into the language files in `app/Language/en-US.ini` at the bottom of the file. Then you can replace the strings with 
`{{ __('text.XXXXXX') }}`